readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -143,6 +143,20 @@ def test_excludes_existing_subprojects(self):
             [''],
         )
 
+    def test_exclude_self_project_as_subproject(self):
+        user = fixture.get(User)
+        project = fixture.get(Project, users=[user])
+        subproject = project
```

Comment:
Do not assign this variable. So, in the form initialization it's clear that we are using the _same_ project for both fields.